http/server.lua

```diff
@@ -1344,6 +1389,10 @@ local exports = {
             hook    = set_hook,
             url_for = url_for_httpd,
 
+            add_connection = add_connection,
+            close_connection = close_connection,
+            check_connections = check_connections,
```

Comment:
Redefinition of those functions looks dangerous. I would not expose them.

Reply:
Redefinition? What do you mean?
It is a definition of internal methods required for testing of idle connections.

Reply:
At least start them from underscore, it'll exclude the fields from autocompletion.

(However if you'll follow my proposal regarding `:read()` timeout -- and I'm sure it is worthful -- those functions will gone.)

Reply:
Non-relevant functions removed (`add_connection`, `close_connection`, `check_connections`).

Reply:
Patch was rewritten (I'm using `timeout` in `tcp_server()` as you suggested before).

Reply:
Now you added the `internal` field instead of the functions. It should be started with underscore to don't participate in tab-completion.

Reply:
Okay.